Document usage of is_exact_match with value (#13895)
DataDog/documentation: content/en/monitors/notify/variables.md

{{/is_exact_match}}
```

> The `is_exact_match` conditional variable also supports [`{{value}}` template variable](#template-variables):

Author: ```suggestion
The `is_exact_match` conditional variable also supports [`{{value}}` template variables](#template-variables):
```

```text
{{#is_exact_match "value" "<VALUE>"}}
  This displays if the value that breached the threshold of the monitor is exactly <VALUE>.
{{/is_exact_match}}
```

> To customize the content of your notification  if the value that breached the threshold of your monitor is 5, use the following:

Author: ```suggestion
To customize the content of your notification if the value that breached the threshold of your monitor is 5, use the following:
```

> To customize the content of your notification if the value that breached the threshold of your monitor is 5, use the following:

Author: ```suggestion
To notify your dev team if the value that breached the threshold of your monitor is 5, use the following:
```

```text
{{#is_exact_match "value" "5"}}

>   This displays if the value that breached the threshold of the monitor is 5. @slack-example

Author: ```suggestion
  This displays if the value that breached the threshold of the monitor is 5. @dev-team@company.com
```

>   This displays if the value that breached the threshold of the monitor is 5.

Author: ```suggestion
  This displays if the value that breached the threshold of the monitor is 5. @slack-example
```
